services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/structure/DigitalCertificatesDirectory.java

```diff
@@ -0,0 +1,60 @@
+package app.coronawarn.server.services.distribution.dgc.structure;
+
+import app.coronawarn.server.common.protocols.internal.dgc.ValueSets;
+import app.coronawarn.server.common.shared.collection.ImmutableStack;
+import app.coronawarn.server.services.distribution.assembly.component.CryptoProvider;
+import app.coronawarn.server.services.distribution.assembly.structure.archive.ArchiveOnDisk;
+import app.coronawarn.server.services.distribution.assembly.structure.archive.decorator.signing.DistributionArchiveSigningDecorator;
+import app.coronawarn.server.services.distribution.assembly.structure.directory.IndexDirectoryOnDisk;
+import app.coronawarn.server.services.distribution.assembly.structure.file.FileOnDisk;
+import app.coronawarn.server.services.distribution.config.DistributionServiceConfig;
+import app.coronawarn.server.services.distribution.config.DistributionServiceConfig.DigitalGreenCertificate;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class DigitalCertificatesDirectory extends IndexDirectoryOnDisk<String> {
+
+  public static final String EXPORT_BIN = "export.bin";
```

Comment:
I dislike this name, as it has to change if the contents changes. How about FILENAME or EXPORT_BINARY_FILENAME or something similar semantical?

Reply:
Right. Updated